app/views/site/update_study_settings.js.erb

```diff
@@ -8,11 +8,7 @@ closeModalSpinner('#update-study-settings-spinner', '#update-study-settings-moda
         $('#study-summary').html("<%= escape_javascript(render partial: 'study_description_view') %>");
         $('#study-download').html("<%= escape_javascript(render partial: 'study_download_data') %>");
         <% if @study.initialized? %>
-          <% if User.feature_flag_for_instance(current_user, 'react_explore')  %>
-            window.SCP.renderExploreView(document.getElementById('study-visualize'), window.SCP.studyAccession)
-          <% else %>
-            $('#study-visualize').html("<%= escape_javascript(render partial: 'study_visualize') %>");
-          <% end %>
+          window.SCP.renderExploreView(document.getElementById('study-visualize'), window.SCP.studyAccession)
```

Comment:
can we now also delete that `study_visualize` partial?

Reply:
I don't think so... looking at the file, it has some functionality in it for IGV/Ideogram.  Likely it can get taken out, but would require refactoring that is out of scope for this particular PR, and already tracked in https://broadworkbench.atlassian.net/browse/SCP-3258.

Reply:
ok, then I'm confused on this change.  If that partial still has needed functionality beyond just calling `renderExploreView`, then shouldn't we be using it here instead of calling `renderExploreView` directly?  It seems like this file should match whatever we do in `study.html.erb`

Reply:
I was correct, originally - we do still need that partial.  It has logic for setting state in regards to various things for the explore tab to render correctly on page load, in addition to the IGV- and Ideogram-specific code.  In [`study.html.erb`](https://github.com/broadinstitute/single_cell_portal_core/blob/development/app/views/site/study.html.erb#L12), we render the `_study_visualize` partial. 

Once we pick up SCP-3258, then I think we could move to remove this partial and make the explore tab 100% React.  But for now, we still need it (I even tried taking it out and calling `renderExploreView` directly but that doesn't work for a couple reasons).